pkg/pillar/cmd/zedrouter/networkinstance.go

```diff
@@ -735,6 +748,31 @@ func doNetworkInstanceModify(ctx *zedrouterContext,
 	}
 }
 
+func checkNIphysicalPort(ctx *zedrouterContext, status *types.NetworkInstanceStatus) error {
+	// check the NI have the valid physical port binding to
+	label := status.Logicallabel
+	if label != "" && !strings.EqualFold(label, "uplink") &&
+		!strings.EqualFold(label, "freeuplink") {
+		var foundMatch bool
+		pub := ctx.subDeviceNetworkStatus
+		items := pub.GetAll()
+		for _, st := range items {
+			devStatus := st.(types.DeviceNetworkStatus)
+			devPort := getDevPort(&devStatus, label)
+			if devPort != nil {
+				foundMatch = true
```

Comment:
Should we break here? And remove the Else part?

Reply:
sure.

Reply:
No need for loop

Reply:
ok